pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -76,12 +77,17 @@ def __init__(self, monitor: str = 'val_loss', min_delta: float = 0.0, patience:
                 log.info(f'EarlyStopping mode set to {self.mode} for monitoring {self.monitor}.')
 
         self.min_delta *= 1 if self.monitor_op == torch.gt else -1
+        self.best_score = torch_inf if self.monitor_op == torch.lt else -torch_inf
```

Comment:
@jeremyjordan This PR #1996 attempted to flip the sign. I think yours here is correct, but wanted to just double check with you.

Reply:
yes, i reverted the change when rebasing with master :)